_maps/RainbowDream/Rainbow_Dream.yaml

```diff
@@ -0,0 +1,251 @@
+  - 1  #  97 Stock rise venture! Increase stock value by 20% in a district of your choice.
+  - 1  #  98 Forced buyout! You can buy a vacant plot or shop for four times its value, whether someone else owns it or not.
+  - 1  #  99 Freebie! What's inside...?
+  - 1  # 100 Freebie! Take a Suit Yourself card!
+  - 0  # 101 Special bonus! Your shops all grow by 21%!
+  - 0  # 102 Moneymaking venture! Roll the die and get 33 times the number shown in gold coins from all other players!
+  - 0  # 103 Misadventure! The values of all your shops drop by 25%!
+  - 0  # 104 Misadventure! You give everyone 80G each!
+  - 0  # 105 Moneymaking venture! Roll the die and get the number shown x your level x 40G from the bank!
+  - 0  # 106 Freebie! Roll the die and get half the number shown of Suit Yourself cards! (Decimals will be rounded down.)
+  - 0  # 107 Revaluation venture! You can expand any one of your shops by 30%.
+  - 1  # 108 Cashback venture! You can sell a shop back to the bank for four times its shop value.
+  - 0  # 109 Revaluation venture! You can expand any one of your shops by 75%.
+  - 0  # 110 Special bonus! You receive 77 times the number of shops you own in gold coins from the bank!
+  - 0  # 111 Cashback venture! You can sell a shop back to the bank for 500G more than its shop value.
+  - 0  # 112 Special bonus! You receive 100 times the number of shops you own in gold coins!
+  - 0  # 113 Moneymaking venture! Roll the die and get the number shown x your level x 20G from the bank!
+  - 0  # 114 Moneymaking venture! Take your level times 40G from all other players!
+  - 1  # 115 Misadventure! All other players can only move forward 7 on their next turn.
+  - 0  # 116 Moneymaking venture! Roll the die and get 60 times the number shown in gold coins from the player in 1st place!
+  - 1  # 117 Adventurous turning point! Everyone gets to choose which way to move on their next go.
+  - 0  # 118 Lucky venture! You get a really big commission until your next turn!
+  - 0  # 119 Misadventure! You give 20% of your ready cash to the player in last place!
+  - 0  # 120 Misadventure! You drop your wallet and lose 20% of your ready cash!
+  - 0  # 121 Capital venture! You can invest up to 400G of the bank's money in your shops.
+  - 0  # 122 Moneymaking venture! The winning player must pay you 20% of their ready cash!
+  - 0  # 123 Dicey adventure! Roll 1/3/5 and your shops expand by 20%. Roll 2/4/6 and everyone else's shops expand by 5%.
+  - 0  # 124 Suit venture! Buy a Suit Yourself card for 50G.
+  - 0  # 125 Dicey adventure! Roll 1/3/5 to warp to a boon square. Roll 2/4/6 to warp to the arcade.
+  - 0  # 126 Revaluation venture! Roll the die and expand your shops by 2% for each number.
+  - 0  # 127 Special arcade adventure! You're invited to play Round the Blocks and Memory Block!
+  - 0  # 128 Special bonus! You receive 55 times the number of shops you own in gold coins from the bank!
+districtNames:
+  en:
```

Comment:
All these district entries can be removed since they are default

Reply:
I know, I didn't remove them from the yaml because I figured it wouldn't matter?